libs/nx-chainlink/src/generators/create-gh-action/files/composite/action.yml.template

```diff
@@ -39,7 +39,7 @@ runs:
     - name: Collect metrics
       if: always()
       id: collect-gha-metrics
-      uses: smartcontractkit/push-gha-metrics-action@031fd89b33b3e7722b0c4d2593ece1861d9f503f
+      uses: smartcontractkit/push-gha-metrics-action@0281b09807758be1dcc41651e44e62b353808c47 # v2.1.0
       with:
         basic-auth: ${{ inputs.gc-basic-auth }}
         hostname: ${{ inputs.gc-host }}
```

Comment:
Missing the org id

Reply:
Fixed in: d0134afbf8a192da1ace4d1f2fcd2a45183b9695